jwst/regtest/conftest.py

```diff
@@ -1,21 +1,22 @@
-from datetime import datetime
 import copy
 import json
 import os
+from datetime import datetime
 from pathlib import Path
 
 import getpass
 import pytest
-from ci_watson.artifactory_helpers import UPLOAD_SCHEMA
+from astropy.io.fits import conf
 from astropy.table import Table
+from ci_watson.artifactory_helpers import UPLOAD_SCHEMA
 from numpy.testing import assert_allclose, assert_equal
-from astropy.io.fits import conf
 
 from jwst.regtest.regtestdata import RegtestData
 from jwst.regtest.sdp_pools_source import SDPPoolsSource
 
 
 TODAYS_DATE = datetime.now().strftime("%Y-%m-%d")
+RELAX_TOL = os.environ.get("RELAX_TOL ", "false") == "true"
```

Comment:
should this have a space at the end of `RELAX_TOL`?

Reply:
omg... 🤦‍♀️ good catch. Lemme fix that and try again.